src/go/privileged-service/pkg/manage/install_windows.go

```diff
@@ -44,12 +44,12 @@ func InstallService(name, displayName, desc string) error {
 	if err != nil {
 		return err
 	}
-	defer m.Disconnect()
+	defer func() { _ = m.Disconnect() }()
```

Comment:
Since we already have a `manage.connect` function, we could have a corresponding `manage.disconnect` function that drops the error and then use `defer disconnect(m)` everywhere (there are 4 instances of this).